mypy_django_plugin/transformers/init_create.py

```diff
@@ -28,8 +28,12 @@ def get_actual_types(ctx: MethodContext | FunctionContext, expected_keys: list[s
 
 
 def typecheck_model_method(
-    ctx: FunctionContext | MethodContext, django_context: DjangoContext, model_cls: type[Model], method: str
-) -> MypyType:
+    ctx: FunctionContext | MethodContext,
+    django_context: DjangoContext,
+    model_cls: type[Model],
+    method: str,
+) -> None:
```

Comment:
This function actually only typecheck and then return `ctx.default_return_type`

I've done this change to decouple typecheking and eventual return type alteration